library/plugins/talaiot-standard/src/test/kotlin/io/github/cdsap/talaiot/OutputPublisherBuildTest.kt

```diff
@@ -33,7 +33,6 @@ class OutputPublisherBuildTest : BehaviorSpec({
                 .build()
             then("logs are shown in the output with the shrugged") {
                 assert(result.output.contains("OutputPublisher"))
-                assert(result.output.contains("¯\\_(ツ)_/¯"))
```

Comment:
CircleCI was failing in this assertion. 
After printing outputs, something changed in terms of encoding:
```
[OutputPublisher]: ?\_(?)_/? ?\_(?)_/? ?\_(?)_/? ?\_(?)_/? ?\_(?)_/?
```
https://app.circleci.com/pipelines/github/cdsap/Talaiot/828/workflows/5fe74f47-e699-4197-b57e-ef0d5ff087b1/jobs/986